web_api.md

```diff
@@ -245,8 +245,47 @@ Instead of `path_to_mets`, one can also use `workspace_id` to specify a workspac
 from the Workspace Server. In case `result_queue_name` property is presented, the result of the processing will be
 pushed to the queue with the provided name. If the queue does not exist yet, it will be created on the fly. This is
 useful when there is another service waiting for the results of processing. That service can simply listen to that queue
-and will be immediately notified when the results are available. The message schema for results can be
-found [here](web_api/result-message.schema.yml), while an example of a result message looks like this:
+and will be immediately notified when the results are available. Below is a simple Python script to demonstrate how a
+service can listen to the `result_queue_name` and act accordingly.
+
+```python
+import pika, sys, os
+
+def main():
+    connection = pika.BlockingConnection(pika.ConnectionParameters(host='localhost'))
```

Comment:
> In general, this would be the `--address` of the server.

No, it is the address of the RabbitMQ. The client connects directly to RabbitMQ to listen to the result queue, and it has nothing to do with Web API in this case.

> Also, what about credentials? The RabbitMq config seems to have username/password or keyfile. But again, how does the Web API client get to know?

Yeah, the client/listener must somehow know the credentials to be able to connect to RabbitMQ, maybe via the administrator, who configured RabbitMQ.

Reply:
> > In general, this would be the `--address` of the server.
> 
> No, it is the address of the RabbitMQ. The client connects directly to RabbitMQ to listen to the result queue, and it has nothing to do with Web API in this case.

What I meant was: since the Processing Server usually deploys RabbitMQ locally, it would be the same host (and perhaps the default / hard-coded port). But otherwise, some mechanism of advertising the queue to all clients (hence `/discovery`) would be needed.

Reply:
Yes, but that is up to the IPs to decide, what they want to return from the `/discovery` endpoint. It's OK to return the address of the RabbitMQ instance, but credentials should not be returned, since the returned data is plain text.

So, in any case, the client must somehow know the address and credentials to connect to RabbitMQ, it can be via the administrator, as I said.

Reply:
Ok, granted, `/discovery` contents should not be overly specified. And since the queue model is optional, and you even have a callback endpoint now, and you amended the code snippet with port and credentials, it should be clear enough now.